rr_computer_vision/include/traffic_light.hpp

```diff
@@ -1,25 +1,53 @@
 /**
  * @file rr_traffic_light.hpp
  * @brief Traffic Light Header File
- * @author Jamie Kim
- * @author Jason Leung
- * @author Toni Ogunmade(oluwatoni)
- * @author Adrian Malaran
- * @competition IARRC 2018
+ * @author Yuchi(Allan) Zhao
+ * @competition IARRC 2019
  */
 
-#ifndef TRAFFIC_LIGHT
-#define TRAFFIC_LIGHT
+#ifndef RR_TRAFFIC_LIGHT
```

Comment:
I like to keep these include guards matching the file name exactly unless there is a reason to add something extra to it. I think it's better to keep this as just TRAFFIC_LIGHT